ds_judgements_public_ui/sass/includes/_phase_banner.scss

```diff
@@ -26,6 +26,17 @@
     font-weight: 700;
   }
 
+  &__beta-phase {
+    background-color: $color-yellow;
+    color: $color-black;
+    text-transform: uppercase;
+    padding: $space-1 $space-1;
+    margin-left: $space-1;
+    vertical-align: $space-1;
```

Comment:
If we style `.service-introduction__header` with flex-box and align everything centrally, we can guarantee this will be central in all devices and in all instances/accessibility settings, i.e.:

```
.service-introduction__header {
    display: flex;
    align-items: center;
    justify-content: center;
    gap: 0.25rem;
}
```

Then this can become:

```
&__beta-phase {
    background-color: $color-yellow;
    color: $color-black;
    text-transform: uppercase;
    padding: $space-1;
    font-weight: 700;
    font-size: $typography-sm-text-size;
}